src/main.cpp

```diff
@@ -2239,16 +2252,17 @@ struct CompareBlocksByHeightMain
     else return(coins.vout[n].nValue);
 }*/
 
-bool myAddtomempool(const CTransaction &tx, CValidationState *pstate, bool fSkipExpiry)
+bool myAddtomempool(const CTransaction &tx, CValidationState *pstate, bool *pMissingInputs, bool fSkipExpiry)
 {
     CValidationState state;
     if (!pstate)
         pstate = &state;
-    CTransaction Ltx; bool fMissingInputs,fOverrideFees = false;
-    if ( mempool.lookup(tx.GetHash(),Ltx) == 0 )
+    CTransaction Ltx; 
+    bool fOverrideFees = false;
+    if (mempool.lookup(tx.GetHash(), Ltx) == false)
     {
         if ( !fSkipExpiry )
-            return(AcceptToMemoryPool(mempool, *pstate, tx, false, &fMissingInputs, !fOverrideFees, -1));
+            return(AcceptToMemoryPool(mempool, *pstate, tx, false, pMissingInputs, !fOverrideFees, -1));
```

Comment:
The overloading of this function changes the default value of`pMissingInputs` from false to uninitialized. Please make sure that's what you intended.

Reply:
i believe it does not really matter as pMissingInputs is an output parameter which is initialised right at the beginning of AcceptToMemoryPool